### PR TITLE
[stable/etcd-operator] chore: updates etcd-operator to v0.9.4

### PR DESCRIPTION
**Pull request by marcostvz**
#### What this PR does / why we need it:
* updates etcd-operator to v0.9.4
* adds securityContext to operators
* uses only one serviceAccount for all operators
* adds imagePullSecrets to serviceAccount

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/etcd-operator/templates/backup-etcd-crd.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {{- if .Values.customResources.createBackupCRD }}
----
```
Comment (alejandroEsc) [inline stable/etcd-operator/templates/backup-etcd-crd.yaml]: Can you leave the initial `---` for now.

File: stable/etcd-operator/templates/backup-operator-deployment.yaml [hunk 2]
```diff
@@ -22,7 +21,7 @@ spec:
         app: {{ template "etcd-backup-operator.fullname" . }}
         release: {{ .Release.Name }}
     spec:
-      serviceAccountName: {{ template "etcd-backup-operator.serviceAccountName" . }}
```
Comment (alejandroEsc) [inline stable/etcd-operator/templates/backup-operator-deployment.yaml:25]: nice I like this.

File: stable/etcd-operator/templates/etcd-cluster-crd.yaml [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {{- if .Values.customResources.createEtcdClusterCRD }}
----
```
Comment (alejandroEsc) [inline stable/etcd-operator/templates/etcd-cluster-crd.yaml]: please leave initial `---` marks for now.

**Review by alejandroEsc — CHANGES_REQUESTED** [review]
LGTM overall, nice job. Will be testing myself soon.

**Comment by marcostvz** [comment]
Hey @alejandroEsc , check my last commits with the changes you requested (and I also modified a couple of things I noticed later). Thanks a lot! :)

**Comment by marcostvz** [comment]
Hello @alejandroEsc , does this PR need anything else? Thanks!

**Comment by alejandroEsc** [comment]
@marcostvz Sorry! been busy here and will look today (promise) @davidkarlsen why wasn't i able to set ok-to-test before?

**Pull request merged**